.github/workflows/check-kernel-config.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# This action checks that required kernel configs have not been removed or
+# modified to an undesirable value. It also checks that new configs added to
+# the kernel config files are in the list of required configs.
+name: Kernel Required Configs Check
+
+on:
```

Comment:
If it's in your time budget for this, I'd recommend adding a trigger for `workflow_dispatch`. This would allow you to [run the workflow manually](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#configuring-a-workflow-to-run-manually), say, in cases where you just want to check the validity of each config independent of any changes.

Reply:
was having trouble with this. Will look into in the future